doc/Makefile

```diff
@@ -16,6 +18,7 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
```

Comment:
Should "new" be removed from the description to get the documentation timeless?

Reply:
That's a good point, @PipKat. This was already included in this `make.bat`. Let me change it.